inc/modules/export/xhtml/class-xhtml11.php

```diff
@@ -1370,7 +1371,7 @@ protected function renderPartsAndChaptersGenerator( $book_contents, $metadata )
 				$chapter_subtitle = trim( get_post_meta( $chapter_id, 'pb_subtitle', true ) );
 				$chapter_author = $this->contributors->get( $chapter_id, 'pb_authors' );
 
-				if ( \Pressbooks\Modules\Export\Export::shouldParseSubsections() && \Pressbooks\Book::getSubsections( $chapter_id ) !== false ) {
+				if ( $parse_subsections && \Pressbooks\Book::getSubsections( $chapter_id ) !== false ) {
```

Comment:
Small improvement to avoid calling the same method for each chapter since this is an export option.